doc/conf.py

```diff
@@ -42,6 +45,7 @@
 pyvista.global_theme.font.label_size = 22
 pyvista.global_theme.font.title_size = 22
 pyvista.global_theme.return_cpos = False
+pyvista.global_theme.show_edges = True
```

Comment:
For https://github.com/MatthewFlamm/pyransame/pull/136#issuecomment-1868364974

If you're not a fan of this, feel free to remove:

```suggestion
```

Reply:
Wait nevermind, this is not propagating to the plot directive 😕